src/main/webapp/style.css

```diff
@@ -1,8 +1,22 @@
+#emphasis {
+  color: violet;
+}
+
 h1 {
```

Comment:
Looking at [the CSS style guide](https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors), it seems like we should be avoiding using type selectors like `h1` for CSS. I also think it's important to remember that this will apply to every `h1` element that uses this stylesheet, which I don't think is what we want. Please consider using [Bootstrap margins](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Reply:
You asked me to use the display-1 from bootstrap for the heading. Because display-1 is pre-defined in bootstrap, I cannot change it to add a top margin. How do you recommend I make a heading which is h1, display-1, and also has a custom margin, without using type selectors?

Reply:
Maybe we could try using an ID for this particular header?

Reply:
Good idea @Obi! Trying that now.

Reply:
You can apply multiple Bootstrap classes to the same element, so I imagine something like this would work:
```
<h1 class="display-1 mt-4">[...]</h1>
```